pkg/apimachinery/apis/common/v0alpha1/unstructured.go

```diff
@@ -63,7 +62,9 @@ func (u *Unstructured) DeepCopy() *Unstructured {
 	}
 	out := new(Unstructured)
 	*out = *u
-	out.Object = runtime.DeepCopyJSON(u.Object)
+
+	jj, _ := json.Marshal(u.Object)
```

Comment:
I kept getting failures in the DeepCopy function -- gave up and tried dumb JSON write then read